guides/common/modules/proc_installing-convert2rhel.adoc

```diff
@@ -0,0 +1,29 @@
+[id="installing-convert2rhel_{context}"]
+
+= Installing Convert2RHEL
+
+The following procedure makes Convert2RHEL available to your system.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* *>* *Products* and click *Create Product*.
+. In the *Name* field, enter the name of the product.
+{Project} automatically completes the *Label* field based on your entry for *Name*.
+. Optional: From the *GPG Key* list, select the GPG key for the product.
+. Optional: From the *SSL CA Cert* list, select the SSL CA certificate for the product.
+. You do not need to complete *SSL Client Cert* or *SSL Client Key*.
+. Optional: From the *Sync Plan* list, select an existing sync plan or click *Create Sync Plan* to create a sync plan for your product requirements.
+. In the *Description* field, enter a description of the product.
+. Click *Save*.
+. Click the *Repositories* tab.
+. To add a repository, click *New Repository*.
+. In the *Name* field, enter the name of the repository.
+. From the *Type* list, set the content type to Yum.
+. Do not restrict the architecture or the OS version.
+. Use https://ftp.redhat.com/redhat/convert2rhel/7/os as the upstream URL.
+. Optional: From the *GPG Key* list, select the GPG key for the product.
+. Optional: From the *SSL CA Cert* list, select the SSL CA certificate for the product.
+. You do not need to complete *SSL Client Cert* or *SSL Client Key*.
+. Click *Save*.
+. Click the checkbox next to the new repository, then click *Sync Now*.
+
+Alternatively, click the name of your repository, and download the RPM from a repository such as https://github.com/oamg/convert2rhel/releases/latest.
```

Comment:
download -> upload, people need download the RPM from the upstream repo and to upload the RPM to the Satellite. The issue with the repo though has been fixed meanwhile, we can drop this whole line now.

Reply:
Line removed.